apps/web/components/ListViewCard/ListViewCard.tsx

```diff
@@ -81,18 +82,20 @@ export const ListViewCard = ({
           >
             {heading}
           </Text>
-          <Box style={{ paddingTop: '4px' }}>
-            <Text
-              variant="eyebrow"
-              color="blue400"
-              truncate={false}
-              title={subHeading}
-              paddingBottom={3}
-              lineHeight="sm"
-            >
-              {subHeading}
-            </Text>
-          </Box>
+          {subHeading && (
+            <Box style={{ paddingTop: '4px' }}>
```

Comment:
Box component has a paddingTop prop which could also be used, or is that perhaps too large of a padding top?

Reply:
Yes IIRC the box padding did not fit properly, Ill double check